File: Front/src/app/dtos/new-user-dto.ts
export class NewUserDto {
    UserName: string;
    UserLastName: string;
    UserBornDay: Date;
    CredentialType: unknown;
    UserDni: string;
    UserEmail: string;
    Password: string;
    State: unknown;

    constructor(
        UserName: string,
        UserLastName: string,
        UserBornDay: Date,
        CredentialType: unknown,
        UserDni: string,
        UserEmail: string,
        Password: string,
        State: unknown
    ) {
        this.UserName = UserName;
        this.UserLastName = UserLastName;
        this.UserBornDay = UserBornDay;
        this.CredentialType = CredentialType;
        this.UserDni = UserDni;
        this.UserEmail = UserEmail;
        this.Password = Password;
        this.State = State;
    }
}

File: Back/ARGENTRADER/Dtos/NewUserDto.cs
using static ARGENTRADER.Repositories.DBRepository;

namespace ARGENTRADER.Dtos
{
    public class NewUserDto
    {
       public string UserName { get; set; }
        public string UserLastName { get; set; }
        public DateTime UserBornDay { get; set; }
        public CredentialType CredentialType { get; set; }
        public string UserDni { get; set; }
        public string UserEmail { get; set; }
        public string Password { get; set; }
        public State State { get; set; }
      
    }
}
